portal/views/auth.py

```diff
@@ -1025,3 +1025,21 @@ def authorize(*args, **kwargs):
         return redirect('/')
     # See "hardwired" note in docstring above
     return True
+
+
+@auth.route('/unauthorized_endpoint')
+@oauth.require_oauth()
```

Comment:
Do we need this decorator for an endpoint that is effectively informational? Worried it'll cause a redirect loop if flask-user is sending unauthenticated users here

Reply:
that's a different config, USER_UNAUTHENTICATED_ENDPOINT.  i walked through the flask-user code and confirmed we only hit this when the user is authenticated (logged-in) but not authorized.